ansible/generate_terraform_register.yml

```diff
@@ -1,18 +0,0 @@
----
-- name: Generate register manifest for terraform
```

Comment:
This still gets referenced here: https://github.com/openregister/deployment#terraform-config-for-a-register